src/main/java/com/geronimo/model/Message.java

```diff
@@ -0,0 +1,32 @@
+package com.geronimo.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Set;
+
+@Data
+@Entity
+@Table(name = "messages")
+public class Message extends AuditedEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
```

Comment:
You have an id field already in AbstractEntity, right?

Reply:
Well, I don't